src/overlays/actors/ovl_En_Dt/z_en_dt.c

```diff
@@ -28,59 +95,714 @@ ActorInit En_Dt_InitVars = {
     /**/ EnDt_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BEB29C = {
-    { COLTYPE_NONE, AT_NONE, AC_NONE, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_2, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_NONE, OCELEM_ON, },
+static u16 sTextIds[] = { 0x2ABD, 0x2ABE, 0x2ABF, 0x2AC0, 0x2ABE, 0x2AC1, 0x2AC2, 0x2AC3, 0x2AC4, 0x2ABB,
+                          0x2ABC, 0x2AC6, 0x2AC7, 0x2AC8, 0x2AC9, 0x2ACA, 0x2ACB, 0x2ACC, 0x2ACD, 0x2ACE,
+                          0x2ACF, 0x2AD0, 0x2AD1, 0x2AC5, 0x2AD3, 0x2AD4, 0x2AD2, 0x0000 };
+
+// The code uses EnDt::textIdIndex as an index into this lookup table
+static s32 sCutsceneFocusTargetTable[] = {
+    /* 00 */ EN_DT_CS_FOCUS_TARGET_BAISEN, /* [2ABD] "Most of the townsfolk already sheltered" */
+    /* 01 */ EN_DT_CS_FOCUS_TARGET_MAYOR,  /* [2ABE] "Ahh...hmm...well..." */
+    /* 02 */ EN_DT_CS_FOCUS_TARGET_MUTO,   /* [2ABF] "You cowards! The moon won't fall" */
+    /* 03 */ EN_DT_CS_FOCUS_TARGET_MUTO,   /* [2AC0] "Isn't that right Mayor?" */
```

Comment:
These aren't exact quotes right? Current project policy is to avoid one-to-one text recreations in the code (you don't have to throw them away entirely we just don't want them to perfectly match)

Reply:
The shorter ones are exact, while most are paraphrased. I can remove them, but this was useful to me as I deciphered the various cutscene dialog and thought it might be helpful to others as well.

Reply:
With text now extracted and pretty easy to lookup in the repo, this might be something we want to discuss again. I'm not sure.
However please no exact, even for short ones.

Reply:
Understood. I've removed the text and replaced them with markers where the individual cutscenes begin and end. Would that be acceptable?